RIOT-OS/RIOT
# boards/nucleo-wl55jc : Initial support

This PR adds initial support for the [NUCLEO-WL55JC](https://www.st.com/en/evaluation-tools/nucleo-wl55jc.html) board. 

### Testing procedure

Tested I/O's and onboard peripherals on a physical board. Known issues : 

1. ~~HSE clock however is not getting steady~~ **Fixed**
2. ~~RTT fails~~  **Fixed**
3. ~~Flash page tests fail~~ **Fixed**
4. HWRNG fails
5. LoRa radio support pending. 

**Note**: Update to latest OpenOCD ( 0.11.0 or later) to flash/debug. Also, make sure that Udev rules are updated.

### Issues/PRs references

~~Depends on PR #16237 . Rebase to the same for testing.~~

HIL Tests : [PR](https://github.com/RIOT-OS/RobotFW-tests/pull/101)
